core/domain/opportunity_services.py

```diff
@@ -654,8 +675,17 @@ def increment_question_counts(skill_ids, delta):
         delta: int. The delta for which to increment each question_count.
     """
     updated_skill_opportunities = (
-        _get_skill_opportunity_with_updated_question_count(skill_ids, delta))
-    _save_skill_opportunities(updated_skill_opportunities)
+        _get_skill_opportunities_with_updated_question_counts(skill_ids, delta))
+    to_be_updated_opportunities = []
+    to_be_deleted_opportunity_ids = []
+    for opp in updated_skill_opportunities:
+        if opp.question_count >= constants.MAX_QUESTIONS_PER_SKILL:
```

Comment:
even after question count reaches a threshold we can have more questions for those skills, right? We can consider an opportunity with the count greater than threshold to be least important, maybe?

Assume a contributor wants to contribute a Fraction question but the fraction question count is more than the threshold, in this case I think we should allow the user to contribute fraction question?

@seanlip @sagangwee Any thoughts?

Reply:
Yes it's OK to contribute more questions even if a skill is already "full".

Reply:
That makes sense. We can leave those question opportunities then.